tests/scenarios/authstore.js

```diff
@@ -0,0 +1,158 @@
+export const scenarios = [
```

Comment:
this isn't currently in use. it can either be deleted or we can just keep it around for later when we finish up the strategy runner approach to authstore tests.